### PR TITLE
Fix lastBlockNumber fetch in wait_until_node_synced_gcp.sh

### PR DESCRIPTION
**Pull request by platfowner**
Change summary:
- Fix lastBlockNumber fetch in wait_until_node_synced_gcp.sh

### CODE REVIEW

File: json_rpc/index.js [hunk 1]
```diff
@@ -64,8 +64,7 @@ module.exports = function getMethods(node, p2pServer, eventHandler, minProtocolV
 
     ain_getRecentBlockNumber: function(args, done) {
```
Comment (platfowner) [inline json_rpc/index.js:65]: @liayoo Just realized that we can use similar names for both json rpc api and dev client api. which one do you prefer among 'last block number' and 'recent block number'?

Comment (liayoo) [reply json_rpc/index.js:65]: Ah, thanks for pointing it out! I like "last block number", since we've been using the term more than "recent block number"~

**Review by liayoo — APPROVED** [review]
LGTM, thanks!

File: json_rpc/index.js (hunk 1 above)
Comment (platfowner) [reply json_rpc/index.js:65]: Got it, I'll make changes so.

Comment (platfowner) [reply json_rpc/index.js]: Done, thanks!

**Pull request merged**